how-to-guides/re-run-failed-training/notebooks/re_run_failed_training.ipynb

```diff
@@ -0,0 +1,540 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i19Y0g3YGvV2"
+   },
+   "outputs": [],
+   "source": [
+    "model = BaseModel(\n",
+    "    failed_run_params[\"input_sz\"],\n",
+    "    failed_run_params[\"input_sz\"],\n",
+    "    failed_run_params[\"n_classes\"],\n",
+    ").to(failed_run_params[\"device\"])\n",
+    "criterion = nn.CrossEntropyLoss()\n",
+    "optimizer = torch.optim.SGD(model.parameters(), lr=failed_run_params[\"lr\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ASgLO3T3W-cw"
+   },
+   "source": [
+    "### Log losses and metrics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "MgGfdruzNB3t",
+    "outputId": "e049c918-0ae3-4143-8f88-b26cdf2084bc"
+   },
+   "outputs": [],
+   "source": [
+    "for i, (x, y) in enumerate(trainloader, 0):\n",
+    "    x, y = x.to(failed_run_params[\"device\"]), y.to(failed_run_params[\"device\"])\n",
+    "    optimizer.zero_grad()\n",
+    "    outputs = model.forward(x)\n",
+    "    _, preds = torch.max(outputs, 1)\n",
+    "    loss = criterion(outputs, y)\n",
+    "    acc = (torch.sum(preds == y.data)) / len(x)\n",
+    "\n",
+    "    new_run[\"training/batch/loss\"].log(loss)\n",
+    "\n",
+    "    new_run[\"training/batch/acc\"].log(acc)\n",
+    "\n",
+    "    loss.backward()\n",
+    "    optimizer.step()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "JhE6C9xv-lXK"
+   },
+   "source": [
+    "# Stop run\n",
+    "\n",
+    "<font color=red>**Warning:**</font><br>\n",
+    "\n",
+    "Once you are done logging, you should stop tracking the run using the [stop()](https://docs.neptune.ai/api-reference/project#.stop) method.\n",
+    "This is needed only while logging from a notebook environment. While logging through a script, Neptune automatically stops tracking once the script has completed execution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Meg02T9p9314",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "failed_run.stop()\n",
+    "new_run.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explore the run in the Neptune UI\n",
+    "\n",
+    "After running the code cell in **Step 4**, you will get a link on the cell output similar to https://app.neptune.ai/common/showroom/e/SHOW-3695 with: \n",
+    "* **common/showroom** replaced by **your_workspace/your_project**,\n",
+    "* **SHOW-3695** replaced by your *Run ID*. \n",
+    "\n",
+    "**Click on the link to open the Run in Neptune UI.**\n",
+    "\n",
+    "![image](https://files.gitbook.com/v0/b/gitbook-x-prod.appspot.com/o/spaces%2F-MT0sYKbymfLAAtTq4-t%2Fuploads%2FnZUUkjpnlr7VhB24Trzv%2Fimage.png?alt=media&token=7944ed6d-cbc5-4edf-9013-04d5e9e6ef9f)\n",
```

Comment:
All files should be hosted in wordpress. Please coordinate with @Patrycja-J

Reply:
As noted in the previous comment, my suggestion is to just remove this for now. The screenshot is outdated and not crucial to understanding the goal.